src/main/java/org/proffart/football/training/api/impl/GroupApiImpl.java

```diff
@@ -0,0 +1,31 @@
+package org.proffart.football.training.api.impl;
+
+import org.proffart.football.training.api.GroupApi;
+import org.springframework.stereotype.Component;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+
+/**
+ * Author Artak Mnatsakanyan
+ * Date 9/13/16
+ * Time 12:04 PM
+ */
+@Component
+@Path("/group")
+public class GroupApiImpl implements GroupApi {
+
+
+    @POST
+    @Override
+    public String createGroup(String name) {
+        return null;
+    }
+
+    @PUT
+    @Override
+    public String joinGroup(Integer playerId) {
```

Comment:
RESTful issue: player id must be detected from path 
for that exists two ways 
 1. @QueryParam("id") Long id  : path?id=123
 2. @Path("/join/{Id}"), @PathParam("id") Long id : path/join/123